test/lib/jdk/test/lib/containers/docker/DockerTestUtils.java

```diff
@@ -321,10 +321,12 @@ private static String limitLines(String buffer, int nrOfLines) {
 
     private static void generateDockerFile(Path dockerfile, String baseImage,
                                            String baseImageVersion) throws Exception {
-        String template =
-            "FROM %s:%s\n" +
-            "RUN apt-get install libubsan1\n" +
-            "COPY /jdk /jdk\n" +
+        String template = "FROM %s:%s\n";
+        boolean isUbsan = Boolean.getBoolean("jdk.test.docker.image.isUbsan");
+        if (baseImage.contains("ubuntu") && isUbsan) {
+            template += "RUN apt-get update && apt-get install -y libubsan1\n";
+        }
```

Comment:
It would be cleaner if we did the property processing in `DockerfileConfig` so that we have it in one place.